internal/component_manager.go

```diff
@@ -95,7 +94,7 @@ func RegisterComponents() {
 // all components listed in the Components array.
 //
 // When it is not possible to register a component,
-// an error will be printed into the log.
+// an error will be printed into the logger.
```

Comment:
Are errors printed into the "_log_" or printed into the "_logger_" this sounds weird, Refactor Issue?